frontend/javascripts/libs/utils.ts

```diff
@@ -407,7 +407,12 @@ export function stringToNumberArray(s: string): Array<number> {
   const result = [];
 
   for (const e of stringArray) {
-    const newEl = parseFloat(e);
+    let newEl;
+    try {
+      newEl = eval(e);
```

Comment:
@daniel-wer I adapted the position input view so that one can also do mathematical expressions in it. This was handy during development (and I also imagine this to be helpful when inspecting datasets analytically). For example, you can change the position 1000, 1000, 1000 to 2 * 1000 + 1234, 1000, 1000 and it will immediately jump to 3234, 1000, 1000.
The implementation with `eval` should be fine security-wise, since the field can only be edited by the user (who has control over JS anyway). The position encoded in the url is **not** eval'ed.
What do you think about this feature and the implementation? Do you think it can stay or should I remove it again?